### PR TITLE
Simplify queue manager

### PR DESCRIPTION
**Pull request by dominikvoda**

### CODE REVIEW

File: src/Jobs/FailResolving/DelayRules/ConstantDelayRule.php [hunk 1]
```diff
@@ -19,8 +19,8 @@ public function __construct(int $constantDelayInSeconds)
     }
 
 
-    public function getDelay(JobInterface $job, Throwable $exception): int
+    public function getDelayInMilliseconds(JobInterface $job, Throwable $exception): int
     {
-        return $this->constantDelayInSeconds;
+        return $this->constantDelayInSeconds * 1000;
```
Comment (milan-zelenka) [inline src/Jobs/FailResolving/DelayRules/ConstantDelayRule.php:24]: Having change like that, what is a point of having ConstantDelayRule and ConstantDelayRuleWithMilliseconds. Wouldn't be more useful to have just ConstantDelayRule which would have property constantDelayInMilliseconds?

**Review by milan-zelenka — APPROVED** [review]

File: src/Queue/RabbitMQ/RabbitMQQueueManager.php [hunk 2]
```diff
@@ -84,55 +77,55 @@ public function consumeMessages(callable $consumer, string $queueName, array $pa
...
+                $this->setUpChannelAndStartConsuming($prefetchCount, $queueName, $noAck, $consumer);
             }
         }
 
         $this->closeConnection();
     }
 
 
-    public function push(JobInterface $job): void
+    private function setUpChannelAndStartConsuming(
```
Comment (michal-tichy-84) [inline src/Queue/RabbitMQ/RabbitMQQueueManager.php:88]: [M] We usually put private methods below the public ones

**Review by michal-tichy-84 — COMMENTED** [review]
[O] Plus maybe a test for 0 delay in `RabbitMQQueueManagerTest`?

**Review by michal-tichy-84 — APPROVED** [review]

**Comment by michal-tichy-84** [comment]
Hmm, confident for 1 review only? :hushed:

**Pull request merged**